run.py

```diff
@@ -73,14 +75,17 @@ def load_data(self):
 		self.triples  = ddict(list)
 
 		for (sub, rel), obj in self.sr2o.items():
-			self.triples['train'].append({'triple':(sub, rel, -1), 'label': self.sr2o[(sub, rel)], 'sub_samp': 1})
-
+				self.triples['train'].append({'triple':(sub, rel, -1), 'label': self.sr2o[(sub, rel)], 'sub_samp': 1})
```

Comment:
Please do not reformat code, as this will make it hard to review it. Could you remove all the formatting differences?